src/main/java/schmidt/core/Schmidt.java

```diff
@@ -0,0 +1,307 @@
+package schmidt.core;
+
+import schmidt.exception.SchmidtException;
+import schmidt.task.Deadline;
+import schmidt.task.Event;
+import schmidt.task.Task;
+import schmidt.task.Todo;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * This is the class for Schmidt, a multi-functional chatbot
+ */
+public class Schmidt {
+    /**
+     * This is the main method which makes use of Schmidt class.
+     *
+     * @param args Unused.
+     */
+    public static void main(String[] args) {
+        String LOGO = "░██████╗░█████╗░██╗░░██╗███╗░░░███╗██╗██████╗░████████╗\n" +
+                "██╔════╝██╔══██╗██║░░██║████╗░████║██║██╔══██╗╚══██╔══╝\n" +
+                "╚█████╗░██║░░╚═╝███████║██╔████╔██║██║██║░░██║░░░██║░░░\n" +
+                "░╚═══██╗██║░░██╗██╔══██║██║╚██╔╝██║██║██║░░██║░░░██║░░░\n" +
+                "██████╔╝╚█████╔╝██║░░██║██║░╚═╝░██║██║██████╔╝░░░██║░░░\n" +
+                "╚═════╝░░╚════╝░╚═╝░░╚═╝╚═╝░░░░░╚═╝╚═╝╚═════╝░░░░╚═╝░░░";
```

Comment:
Maybe this could be declare as a constant variable? Also, maybe you could consider moving it out as a class variable?